SDPi_Supplement/asciidoc/volume1/use-cases/tf1-ch-c-use-case-ddes.adoc

```diff
@@ -0,0 +1,57 @@
+// = Use Case Feature: Device Data to Enterprise Systems (DDES)
+
+[sdpi_offset=5]
+=== Use Case Feature {var_use_case_id}: Device Data to Enterprise Systems (DDES)
+
+==== Narrative:
+
+Mercy Hospital is in the middle of a new EHR (Epic, Cerner, etc.) rollout.  Joe Furst is responsible for integrating data from their ICU devices (patient monitors, ventilators, infusion devices, etc.) with the new EHR.  Once they are done, the data from the devices will be reviewed/validated by the ICU clinicians and then automatically entered into the patient’s clinical record.
+
+==== Technical View
+
+.Device Data to Enterprise Systems (DDES) -- Technical View
+
+image::../../images/vol1-diagram-use-case-ddes-tech-view.svg[]
+
+==== Technical Pre-Conditions
+
+*Given* All devices communicate using SDC
+
+*And* There is at least one Data Gateway Service
+
+*And* All devices report either a device label and/or location
+
+*And* A Data Gateway is associated with a specific set of device labels, and/or location(s) (i.e. devices in scope).
+
+==== Scenarios
+
+===== Scenario: DDES {var_use_case_id}.1 - New in scope device is connected to network with MGW service
+
+*Given* Data Gateway service has detected a new in scope device
+
+*When* the Data Gateway service is operational
+
+*Then* the Data Gateway service will connect to the device and export data to the EHR using the HL7 v2 based IHE DEV DOR actor of the DEV DEC profile
```

Comment:
Does it make sense to link ` HL7 v2 based IHE DEV DOR actor of the DEV DEC profile` here?

Reply:
Added a #TODO to the section pointing out the need to standardize how this should be done + to sync with TF-2 Appendix B that includes V2 Gateway specs & links to DEC  and DOR / DOC actors.  Added to the 11 November Agenda for discussion.  Another "where" question + the Appendix B references section already includes the 2019 PCD TF references.